doc/whats_new/v0.24.rst

```diff
@@ -234,10 +234,6 @@ Changelog
   files downloaded or cached to ensure data integrity.
   :pr:`14800` by :user:`Shashank Singh <shashanksingh28>` and `Joel Nothman`_.
 
-- |Feature| :func:`datasets.fetch_openml` now validates md5checksum of arff
```

Comment:
Oh you should let this line here. (It was this pasted entry in 1.0 that needed to be removed).

Reply:
I took it out since it seemed like it was a duplicate. I can add it back in.

Reply:
@glemaitre this is a duplicate of the entry above it, which I guess you didn't notice. So this removal IMO is the correct thing to do (albeit not related to this PR)

Reply:
This is correct. I did not see the duplicate. Good catch then.